### PR TITLE
[JENKINS-69777] Prepare for icon removal from core

### PR DESCRIPTION
**Pull request by martinbirks**
Preparation for core removing sunset icons: https://github.com/jenkinsci/jenkins/pull/5778

Fixes issue https://issues.jenkins.io/browse/JENKINS-69777

### CODE REVIEW

File: src/main/resources/hudson/plugins/matrix_configuration_parameter/MatrixCombinationsParameterValue/rebuild.groovy [hunk 1]
```diff
@@ -70,6 +70,7 @@ private void drawTableBall(Combination combination,AxisList axes,matrixValue,Mat
     result = matrixValue.combinationExists(axes, combination);
     if (run != null && result){
         a(href:rootURL+"/"+run.getUrl()){
+            l.icon(class:"icon-md "+run.getIconColor().getIconClassName())
             img(src: "${imagesURL}/24x24/"+run.getBuildStatusUrl());
```
Comment (cs96and) [inline src/main/resources/hudson/plugins/matrix_configuration_parameter/MatrixCombinationsParameterValue/rebuild.groovy:74]: Shouldn't the `img()` call be removed here?

**Comment by ikedam** [comment]
Looks affected since [jenkins-2.333](https://github.com/jenkinsci/jenkins/releases/tag/jenkins-2.333) and 2.346.1. 
`grey.gif` was removed. (somehow other files like `red.gif`, `blue.gif` and so on are still available)

**Comment by ikedam** [comment]
I verified that `img` `gif` `png` are no longer left in codes.

**Review by ikedam — APPROVED** [review]
Looks good to me.
Thanks!
I'll cut a new release in the next weekend.

**Comment by ikedam** [comment]
matrix-combinations-parameter-1.3.2 with this fix was released.

**Pull request merged**